### ISSUE TITLE
nodejs-12.19.0-1nodesource.x86_64.rpm: Package does not match intended download

### ISSUE THREAD

**Issue by jpswade** [nodesource/distributions#1126]
```
$ sudo yum -y install nodejs
Loaded plugins: extras_suggestions, langpacks, priorities, update-motd
amzn2-core                                                                                                                                                                                                      | 2.4 kB  00:00:00     
amzn2extra-docker                                                                                                                                                                                               | 1.8 kB  00:00:00     
amzn2extra-nginx1                                                                                                                                                                                               | 1.8 kB  00:00:00     
amzn2extra-php7.4                                                                                                                                                                                               | 1.8 kB  00:00:00     
nodesource                                                                                                                                                                                                      | 2.5 kB  00:00:00     
(1/10): amzn2-core/2/x86_64/group_gz                                                                                                                                                                            | 2.5 kB  00:00:00     
(2/10): amzn2-core/2/x86_64/updateinfo                                                                                                                                                                          | 203 kB  00:00:00     
(3/10): amzn2extra-nginx1/2/x86_64/primary_db                                                                                                                                                                   |  14 kB  00:00:00     
(4/10): amzn2extra-php7.4/2/x86_64/updateinfo                                                                                                                                                                   |   69 B  00:00:00     
(5/10): amzn2extra-docker/2/x86_64/updateinfo                                                                                                                                                                   |   69 B  00:00:00     
(6/10): amzn2extra-nginx1/2/x86_64/updateinfo                                                                                                                                                                   |   69 B  00:00:00     
(7/10): amzn2extra-php7.4/2/x86_64/primary_db                                                                                                                                                                   |  88 kB  00:00:00     
(8/10): amzn2extra-docker/2/x86_64/primary_db                                                                                                                                                                   |  65 kB  00:00:00     
(9/10): nodesource/x86_64/primary_db                                                                                                                                                                            |  46 kB  00:00:00     
(10/10): amzn2-core/2/x86_64/primary_db                                                                                                                                                                         |  40 MB  00:00:00     
Resolving Dependencies
--> Running transaction check
---> Package nodejs.x86_64 2:12.19.0-1nodesource will be installed
--> Finished Dependency Resolution

Dependencies Resolved

=======================================================================================================================================================================================================================================
 Package                                            Arch                                               Version                                                            Repository                                              Size
=======================================================================================================================================================================================================================================
Installing:
 nodejs                                             x86_64                                             2:12.19.0-1nodesource                                              nodesource                                              22 M

Transaction Summary
=======================================================================================================================================================================================================================================
Install  1 Package

Total download size: 22 M
Installed size: 68 M
Downloading packages:
nodejs-12.19.0-1nodesource.x86 FAILED                                          
https://rpm.nodesource.com/pub_12.x/el/7/x86_64/nodejs-12.19.0-1nodesource.x86_64.rpm: [Errno -1] Package does not match intended download. Suggestion: run yum --enablerepo=nodesource clean metadata  0.0 B/s |    0 B  --:--:-- ETA 
Trying other mirror.


Error downloading packages:
  2:nodejs-12.19.0-1nodesource.x86_64: [Errno 256] No more mirrors to try.


```

**Comment by davidson-joseph** [comment]
The 12.19.0 package that was just released today fails to install on Amazon Linux 2 containers.
My workaround for now is to run `yum install nodejs-12.18.4-1nodesource` (the previous version before 12.19.0)

**Comment by igsu** [comment]
@jpswade @davidson-joseph thanks for reporting, there was a problem related to the cache of this version which has already been fixed, please try a new clean installation.

**Comment by zach-lum-fm** [comment]
@igsu still seeing this error

**Comment by zach-lum-fm** [comment]
Looks fixed 👍  Thanks @igsu

**Comment by jonmelia** [comment]
We are still experiencing this issue...  
nodesource/pub_12.x/el/7/x86_64/nodejs-12.19.0-1nodesource.x86_64.rpm: [Errno -1] Package does not match intended download. Suggestion: run yum --enablerepo=nodesource clean metadata\nTrying other mirror.\n\n\nError downloading packages:\n  2:nodejs-12.19.0-1nodesource.x86_64: [Errno 256] No more mirrors to try.\n\n",

**Comment by daniel-white** [comment]
@igsu we are still experiencing issues installing this on amazon linux 2. please advise.

**Comment by levshaun-amzn** [comment]
This is still happening to me. I created a new EC2 instance and am getting an identical error to the original poster.

**Comment by wexo-team** [comment]
@igsu This is indeed still an issue actively happening. We're experiencing it for most instances on our EC2 instances running Amazon Linux, but not all. Unfortunately this affects our availability as we depend on node installation while deploying new instances.

**Comment by jonmelia** [comment]
We are also still experiencing this issue.  We had the item ached in our local artifactory and thought that was the issue.  But removing the cached items and even deleting the nodesource repo and re-creating it still hasn't resolved the problem.  We have reverted to using 12.18 until this issue is resolved.

**Comment by igsu** [comment]
We are actively working to solve this. 
It would be helpful to know in which AWS regions you are running these instances.

**Comment by jonmelia** [comment]
Hi Ivan,

We are running in EU-WEST-2

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by jpswade** [comment]
eu-west-1

**Comment by robtpaton** [comment]
eu-west-1 also

**Comment by yezen-alnafei** [comment]
eu-west-1 +1

**Comment by jonmelia** [comment]
This seems to have been resolved since a new package was updated for 12.19 on the 29-Oct-2020 11:36
